### PR TITLE
download chart functionality 

### PR DESCRIPTION
**Pull request by spiderxm**
- [x] Download chart functionality added

### CODE REVIEW

**Comment by spiderxm** [comment]
@dhirensr @Ali-Razmjoo please review

**Review by dhirensr — APPROVED** [review]
looks good!

File: web/static/index.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
     <!-- main.js used as general js file for index.html -->
     <script src="/js/main.js"></script>
     <script src="/js/explore.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (dhirensr) [inline web/static/index.html:29]: @spiderxm : why do we need this ?

Comment (spiderxm) [reply web/static/index.html:29]: I forgot to remove it. Previously I was trying to use font-awesome icon. Will remove it.

Comment (spiderxm) [reply web/static/index.html]: removed

**Pull request merged**